src/index.html

```diff
@@ -8,6 +8,12 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="star--1"></div>
+      <div class="star--2"></div>
+      <div class="star--3"></div>
```

Comment:
each of these should also have `stars` class

```suggestion
      <div class="stars stars--2"></div>
      <div class="stars stars--3"></div>
```